app/src/main/java/net/skyscanner/backpack/demo/stories/NavBarStory.kt

```diff
@@ -56,6 +52,7 @@ class NavBarStory : Story() {
     infix fun of(fragmentLayout: Int) = NavBarStory().apply {
       arguments = Bundle()
       arguments?.putInt(LAYOUT_ID, fragmentLayout)
+      arguments?.putBoolean(SCROLLABLE, false)
```

Comment:
Instead of temporarily blocking component_detail_container scrolling we now use this new API to not put the content in the scroll view.